src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Controllers/BankIdApiController.cs

```diff
@@ -75,9 +75,14 @@ public class BankIdApiController : Controller
         [HttpPost("Initialize")]
         public async Task<ActionResult<BankIdLoginApiInitializeResponse>> Initialize(BankIdLoginApiInitializeRequest request)
         {
+            if (request.LoginOptions == null)
+            {
+                throw new ArgumentNullException(nameof(request.LoginOptions));
+            }
+
             var unprotectedLoginOptions = _loginOptionsProtector.Unprotect(request.LoginOptions);
 
-            SwedishPersonalIdentityNumber personalIdentityNumber;
+            SwedishPersonalIdentityNumber? personalIdentityNumber;
```

Comment:
I suppose that here we expect null from unprotectedLoginOptions?

Reply:
Yes, that can be null if not PIN is passed, which is the default case.